base-2018/partials/_footer.twig

```diff
@@ -0,0 +1,28 @@
+{# Footer #}
+<footer role="contentinfo" class="footer">
+    <div class="container">
+        <nav class="level" role="navigation" aria-label="footer navigation">
+            <div class="level-left">
+                <div class="level-item">
+                    <p>
+                        &copy; 2018 &bullet;
+                        {{ __("This website is Built with <a href='%bolt%' target='_blank' title='Sophisticated, lightweight & simple CMS'>Bolt</a> and <a href='%bulma' target='_blank' title='Free and open source CSS framework based on Flexbox'>Bulma</a>.",{'%bolt%':'http://bolt.cm','%bulma':'http://bulma.io'}) }}
+                    </p>
+                </div>
+            </div>
+            <div class="level-right">
+                {{ menu(
+                    identifier = 'main',
+                    template = 'partials/_sub_menu_footer.twig',
+                    params = {'withsubmenus': false}
+                ) }}
+            </div>
+        </nav>
+        {{ widgets('footer') }}
+    </div>
+</footer>
+
+<script src="{{ asset('js/jquery.js', 'theme') }}"></script>
```

Comment:
[Performance/Optimization] Concat and minify these four lines?

Reply:
Maybe. I kinda like that people can _remove_ stuff if they don't like it. That said: magnific and brightness _do_ need to go into app.js, and I _might_ want to add Prism, for out-of-the-box pretty code snippets.

Reply:
We could do something sensible, like realise that jQuery shouldn't be used post-2015, no?